docker/Config.default.toml

```diff
@@ -24,3 +28,12 @@ log-level = "info"
 # relative path will be resolved relative to Config.toml
 # absolute path will work as is
 path = "./state"
+
+[workers]
```

Comment:
Section itself is optional too, right?

```suggestion

# [workers]
```

Reply:
Yes, it is optional, but I decided to leave it as someone may uncomment an option without uncommenting the section.